src/lib/Language/Coq/FreeVars.hs

```diff
@@ -118,22 +110,22 @@ instance HasBV Qualid (Qualid, [Pattern]) where
   bvOf (con, pats) = fvOf' con <> foldMap bvOf pats
 
 instance HasBV Qualid Sentence where
-  bvOf (AssumptionSentence assum      ) = bvOf assum
-  bvOf (DefinitionSentence def        ) = bvOf def
-  bvOf (InductiveSentence  ind        ) = bvOf ind
-  bvOf (FixpointSentence   fix        ) = bvOf fix
-  bvOf (ProgramSentence   sen    _    ) = bvOf sen
-  bvOf (AssertionSentence assert _pf  ) = bvOf assert
-  bvOf (ModuleSentence        _mod    ) = mempty
-  bvOf (ClassSentence         cls     ) = bvOf cls
-  bvOf (ExistingClassSentence name    ) = fvOf' name
-  bvOf (RecordSentence        rcd     ) = bvOf rcd
-  bvOf (InstanceSentence      ins     ) = bvOf ins
-  bvOf (NotationSentence      notation) = bvOf notation
-  bvOf (LocalModuleSentence   lmd     ) = bvOf lmd
-  bvOf (SectionSentence       sec     ) = bvOf sec
-  bvOf (ArgumentsSentence     _arg    ) = mempty
-  bvOf (CommentSentence       com     ) = fvOf' com
+  bvOf (AssumptionSentence assum) = bvOf assum
+  bvOf (DefinitionSentence def) = bvOf def
+  bvOf (InductiveSentence ind) = bvOf ind
+  bvOf (FixpointSentence fix) = bvOf fix
+  bvOf (ProgramSentence sen _) = bvOf sen
+  bvOf (AssertionSentence assert _pf) = bvOf assert
+  bvOf (ModuleSentence _mod) = mempty
+  bvOf (ClassSentence cls) = bvOf cls
+  bvOf (ExistingClassSentence name) = fvOf' name
+  bvOf (RecordSentence rcd) = bvOf rcd
+  bvOf (InstanceSentence ins) = bvOf ins
+  bvOf (NotationSentence notation) = bvOf notation
+  bvOf (LocalModuleSentence lmd) = bvOf lmd
+  bvOf (SectionSentence sec) = bvOf sec
+  bvOf (ArgumentsSentence _arg) = mempty
+  bvOf (CommentSentence com) = fvOf' com
```

Comment:
The new formatting seems to break the alignment of equality signs in some cases like here. I'm not sure what causes this because further below this does not happen. In some cases the new formatting creates the correct alignment for equality signs where the old tool failed.
```suggestion
  bvOf (AssumptionSentence assum)     = bvOf assum
  bvOf (DefinitionSentence def)       = bvOf def
  bvOf (InductiveSentence ind)        = bvOf ind
  bvOf (FixpointSentence fix)         = bvOf fix
  bvOf (ProgramSentence sen _)        = bvOf sen
  bvOf (AssertionSentence assert _pf) = bvOf assert
  bvOf (ModuleSentence _mod)          = mempty
  bvOf (ClassSentence cls)            = bvOf cls
  bvOf (ExistingClassSentence name)   = fvOf' name
  bvOf (RecordSentence rcd)           = bvOf rcd
  bvOf (InstanceSentence ins)         = bvOf ins
  bvOf (NotationSentence notation)    = bvOf notation
  bvOf (LocalModuleSentence lmd)      = bvOf lmd
  bvOf (SectionSentence sec)          = bvOf sec
  bvOf (ArgumentsSentence _arg)       = mempty
  bvOf (CommentSentence com)          = fvOf' com
```

Reply:
Floskell does not align the equal signs here because the line

```haskell
bvOf (ClassSentence cls)            = bvOf cls
```

a padding of 11 spaces would have to be inserted. However Floskell inserts at most 10 spaces in its default configuration. Maybe we can find a better threshold.

Reply:
I've increased the limit to 20 spaces (see 8532149076645b0db84c95c343e07e3f25f4da53).